app/views/activities/_comment.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if comment %>
+  <% title = "#{comment.commentable.class.to_s.capitalize} ##{comment.commentable.id}" %>
+  <% if comment.commentable.title? %>
+    <% title = comment.commentable.title %>
+  <% end %>
+  <%= presenter.verb %> a comment on <%= link_to title, polymorphic_path([@project, comment.commentable], anchor: dom_id(comment)) %>.
```

Comment:
How easy would it be to change this so instead of saying "Person X created a comment on Issue Y", it said "Person X commented on Issue Y"? The first one isn't wrong but it's a slightly weird way to say it.

I realise we're a bit restricted by the need to make the activities partials/presenters as reusable as possible, if making this change is going to take more than a few minutes then I wouldn't bother.

Reply:
Also, the link here is a tad confusing:

<img width="413" alt="screen shot 2018-07-18 at 11 05 48" src="https://user-images.githubusercontent.com/2656485/42875058-995dcb6c-8a7a-11e8-9773-3a5c20b9c16f.png">

When I view this I'm going to think "okay, what do I have to click to see the comment"? It's not obvious that the link "Issue 2" is a link to *the specific comment* (i.e. it has an anchor tag in the URL) rather than a link to the issue in general.

A more intuitive way to do it, I think, would be for the text to say "admin created [a comment](#) on [Issue 2](#)", where the first link has the anchor tag and the second one doesn't. Or if we change the wording as per my comment above: "admin [commented](#) on [Issue 2](#)"

Reply:
Done this, but:

> I realise we're a bit restricted by the need to make the activities partials/presenters as reusable as possible

This is why I opted for "Person X added a [link:comment] to [link:issue]", so the update and create cases are identical.

Reply:
👍